test/vmtester/tests.cpp

```diff
@@ -64,19 +64,19 @@ TEST_F(evmc_vm_test, set_option_unknown)
 {
     if (vm->set_option)
     {
-        int r = vm->set_option(vm, "unknown_option_csk9twq", "v");
-        EXPECT_EQ(r, 0);
+        enum evmc_set_option_result r = vm->set_option(vm, "unknown_option_csk9twq", "v");
+        EXPECT_NE(r, EVMC_SET_OPTION_SUCCESS);
```

Comment:
The result should be "invalid name".

Reply:
I had those here before, but figured it may better negating the expectation.